### ISSUE TITLE
This might be a small bug in the calculation model

### ISSUE THREAD

**Issue by schittli** [Rem0o/FanControl.Releases#868]
Hello
**Thank you very much for sharing your great work!**

I guess that there is a bug in the calculation model, I have created a simple configuration:
- The GPU temperature is monitored. Because there is no particular GPU load, the temperature is stable.
- I use a linear curve to control the GPU. It is assigned to one GPU Fan

This looks like a bug:
(1) The GPU temperature has 38°C
(2) The linear curve has a minimum of 40°C and 10% min. speed
(3) therefore, the Linear Curve calculates a Fan Speed of 10%
(4) But FanControl set 0% as the effective Fan speed

![image](https://user-images.githubusercontent.com/8282673/173239150-3cdbae6f-1f48-44bc-8ad5-da84f8bfddcd.png)

Thanks a lot, kind regards,
Thomas

**Comment by Rem0o** [comment]
Set your control card to manual mode. Slide the slider arround. My guess is that the control doesn't work at all.   Do you have MSI after burner or something similar opened?

**Comment by schittli** [comment]
Thank you very much for your support!
you're right, a Gigabyte GPU Tweak Tool also controlled the Fan speed.
Therefore, I was able to control the Fan speed with FanControl, but the Gigabyte Tool changed it every some seconds, too.

That gives me an idea:
When FanControl sets a new fan speed, it could _optionally_ check if the fan still has the previously set speed.
If the speed has been changed, then the user probably overlooked other SW and FanSpeed could mark the widget with a warning 😃 

Thanks a lot, kind regards,
Thomas
